### PR TITLE
timed backup script with backup time

### PR DESCRIPTION
**Pull request by migasQ**
- fixed backup time
- dynamic sleep based on duration of previous backup

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -4,16 +4,17 @@ version: '3'
 services:
 
   zammad-backup:
-    command: ["zammad-backup"]
+    command: ["zammad-timed-backup"]
```
Comment (monotek) [inline docker-compose.yml:7]: please keep old command

File: scripts/backup.sh [hunk 2]
```diff
@@ -38,6 +38,23 @@ function zammad_backup {
   echo "backup finished :)"
 }
 
+if [ "$1" = 'zammad-timed-backup' ]; then
```
Comment (monotek) [inline scripts/backup.sh:41]: lets make that the default.
so command name should be "zammad-backup"

**Review by monotek — CHANGES_REQUESTED** [review]
please fix ci

File: scripts/backup.sh [hunk 3]
```diff
@@ -11,7 +11,7 @@ set -e
 : "${POSTGRESQL_DB:=zammad_production}"
 
 function check_railsserver_available {
-  until (echo > /dev/tcp/${ZAMMAD_RAILSSERVER_HOST}/${ZAMMAD_RAILSSERVER_PORT}) &> /dev/null; do
+  until (echo > "/dev/tcp/$ZAMMAD_RAILSSERVER_HOST/$ZAMMAD_RAILSSERVER_PORT") &> /dev/null; do
```
Comment (monotek) [inline scripts/backup.sh:14]: why was this changed?

**Comment by migasQ** [comment]
Because the bash script linter complained about this line. I can reverse it but then the linter will keep failing. Am 12.05.2023 um 17:41 schrieb André Bauer ***@***.***>:﻿
@monotek commented on this pull request.



In scripts/backup.sh:
> @@ -11,7 +11,7 @@ set -e
 : "${POSTGRESQL_DB:=zammad_production}"
 
 function check_railsserver_available {
-  until (echo > /dev/tcp/${ZAMMAD_RAILSSERVER_HOST}/${ZAMMAD_RAILSSERVER_PORT}) &> /dev/null; do
+  until (echo > "/dev/tcp/$ZAMMAD_RAILSSERVER_HOST/$ZAMMAD_RAILSSERVER_PORT") &> /dev/null; do

why was this changed?

**Review by monotek — APPROVED** [review]

**Pull request merged**